kfp_notebook/pipeline/_notebook_op.py

```diff
@@ -112,13 +112,13 @@ def __init__(self,
                                  )
 
             if self.pipeline_inputs:
-                argument_list.append(['--inputs', self.pipeline_inputs])
+                argument_list.append('--inputs "{}" '.format(self.pipeline_inputs))
 
             if self.pipeline_outputs:
-                argument_list.append(['--outputs', self.pipeline_outputs])
+                argument_list.append('--outputs "{}" '.format(self.pipeline_outputs))
 
             kwargs['command'] = ['sh', '-c']
-            kwargs['arguments'] = argument_list
+            kwargs['arguments'] = "".join(argument_list)
```

Comment:
I believe this is supposed to be one big string, instead of a multi-entry list, thus why I made these changes above. Note that for testing I sort of reverted the previous change on Elyra side and validated the bootstrap command line on the KFP side. Anyway, we need better docs and tests on these things.